setup.py

```diff
@@ -2,18 +2,24 @@
 
 setup(
     name="contessa",
-    version="0.1.4",
+    version="0.1.5",
```

Comment:
this version is tied to migration in migration map, so maybe it would be better to change it now

Reply:
we should change this version now, because in case of version 0.1.4. package will be not possible to use. This is also prevention from forgetting to change version.